tools/src/main/scala/scala/scalanative/codegen/ResourceEmbedder.scala

```diff
@@ -49,9 +64,13 @@ private[scalanative] object ResourceEmbedder {
                   (pathString, path)
                 }
 
-              if (isInIgnoredDirectory(path)) {
+              if (shouldIgnore(path)) {
                 config.logger.debug(
-                  s"Did not embed: $pathName - file in the ignored \'scala-native\' folder."
+                  s"Did not embed: $pathName - " +
+                    s"Include patterns: ${config.compilerConfig.resourceIncludePatterns
+                        .mkString(", ")}; " +
+                    s"Exclude patterns: ${(config.compilerConfig.resourceExcludePatterns :+ ScalaNativeExcludePattern)
+                        .mkString(", ")}"
```

Comment:
This part could be cached as it needs to be computed only once, so it can be appended to `s"Did not embed: $pathName - "` 
But, even better we can refactor it out check which exactly pattern was applied to exclude given path. So we'd have something like this 
```
 val notInIncludePatterns = s"Not matched by any include path ${includePatterns.mkString}"
 type IgnoreReason = String
 def shouldIgnore(path: Path): Option[IgnoreReason] =  
    includePatterns.find(_.matches(path)).fold(notInIncludePatterns){ includePattern => 
     excludePatterns.find(_.matches(path).map(excludePattern => s"Matched by $includePattern by excluded by $excludePattern")
     
  ... 
  shouldIgnore(path) match {
  case Some(reason) => config.logger.debug("Did not embed: $path - $reason")
  case None =>  ??? 
  }